pkg/common/common.go

```diff
@@ -315,3 +316,30 @@ func ACHeadersHandler(allowedMethods ...string) mux.MiddlewareFunc {
 		})
 	}
 }
+
+// GetRepoRefference returns the components of a repoName:tag or repoName@digest string. If the format is wrong
```

Comment:
This should move to pkg/common/utils.go together with the other similar functions.